src/widgets/Components/LastNews.tsx

```diff
@@ -0,0 +1,186 @@
+import { useTheme } from "@react-navigation/native";
+import { Newspaper, Clock8, Paperclip } from "lucide-react-native";
+import React, { forwardRef, useEffect, useImperativeHandle, useMemo } from "react";
+import { Text, View } from "react-native";
+import {NativeText} from "@/components/Global/NativeComponents";
+import { WidgetProps } from "@/components/Home/Widget";
+import { useCurrentAccount } from "@/stores/account";
+import {useNewsStore} from "@/stores/news";
+import {updateNewsInCache} from "@/services/news";
+import formatDate from "@/utils/format/format_date_complets";
+import {error as log_error} from "@/utils/logger/logger";
```

Comment:
le contenu des actualités est du HTML, à décrypter

```suggestion
import {error as log_error} from "@/utils/logger/logger";
import parse_news_resume from "@/utils/format/format_pronote_news";
```